server/src/test/java/io/crate/rest/action/SqlHttpHandlerTest.java

```diff
@@ -47,6 +31,25 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+import org.elasticsearch.common.breaker.NoopCircuitBreaker;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.http.netty4.cors.Netty4CorsConfigBuilder;
+import org.junit.Test;
+
+import io.crate.action.sql.SQLOperations;
+import io.crate.action.sql.Session;
+import io.crate.action.sql.SessionContext;
+import io.crate.auth.AccessControl;
+import io.crate.auth.AuthSettings;
+import io.crate.user.User;
+import io.crate.user.UserLookup;
+import io.netty.handler.codec.http.DefaultHttpHeaders;
+import io.netty.handler.codec.http.FullHttpRequest;
+
 public class SqlHttpHandlerTest {
 
     class SameNameUserLookup implements UserLookup {
```

Comment:
I think we don't need this anymore (was introduced by the prev change in this area) - wherever we use   
`new SameNameUserLookup()` we can use  `() -> List.of(User.of(name_used_in_test_case))`